tests/unit/test_base_command.py

```diff
@@ -38,6 +40,23 @@ class Test_base_command_logging(object):
     options
     """
 
+    def setup(self):
+        self.old_time = time.time
+        time.time = lambda: 1547704837.4
+        self.old_tz = getattr(os.environ, 'TZ', None)
+        os.environ['TZ'] = 'UTC'
```

Comment:
Would it be worth adding a comment saying why you need to modify the environment here (I'm guessing it's because the displayed timestamp is in the local timezone)?